src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -833,8 +872,16 @@ void FitsSchemaInputMapper::readRecord(BaseRecord &record, afw::fits::Fits &fits
             record.set(_impl->flagKeys[bit], _impl->flagWorkspace[bit]);
         }
     }
-    for (auto iter = _impl->readers.begin(); iter != _impl->readers.end(); ++iter) {
-        (**iter).readCell(record, row, fits, _impl->archive);
+    if (N_ROWS_TO_PREP != 1 && row % N_ROWS_TO_PREP == 0) {
+        // Give readers a chance to read and cache up to N_ROWS_TO_PREP rows-
+        // worth of values.
```

Comment:
Do we have tests that read both more and less than `N_ROWS_TO_PREP`, to ensure this is working>

Reply:
We definitely have tests that read less.  I'll wait until @erykoff's tests for the right value of `N_ROWS_TO_PREP` to see if I should look into adding one that reads more.

Reply:
I don't think I have any more tests planned than what I posted on the ticket: 32 is better for a typical `calexp_src` file (wide/short), and 256 is better for a giant table (narrow/tall).